scss/components/popover.scss

```diff
@@ -270,3 +270,8 @@ $block: #{$fd-namespace}-popover;
         }
     }
 }
+
```

Comment:
We should remove the `[aria-hidden="true"]` selector on line 151.

Is `.popover-body__pgd` just for the playground? If so, it should live in https://github.com/SAP/fundamental-styles/blob/master/test/resources/layout.css

Reply:
@jbadan thank you, will do. Wasn't quite sure in which file it should be.